weblate_web/tests.py

```diff
@@ -1309,6 +1312,73 @@ def test_hosted_pay_yearly(self):
             timezone.now().date() + timedelta(days=3) + relativedelta(years=1),
         )
 
+    @override_settings(ZAMMAD_TOKEN="test")  # noqa: S106
+    @responses.activate
+    def test_decicated_new(self):
+        mock_vies()
+        self.create_packages()
+        responses.add(
+            responses.POST,
+            "https://care.weblate.org/api/v1/tickets",
+            json={
+                "id": 19,
+                "group_id": 2,
+                "priority_id": 2,
+                "state_id": 1,
+                "organization_id": None,
+                "number": "22019",
+                "title": "Help me!",
```

Comment:
I am not sure this won’t end in the created ticket. @nijel?

Reply:
No, this is just a mock used for tests.